apps/aot/src/app/leaderboard/getAotChallengeIds.ts

```diff
@@ -4,32 +4,32 @@ import { getCurrentAdventDay } from '~/utils/time-utils';
 
 export const getAotChallengeIdsSoFar = async () => {
   const currentAdventDay = getCurrentAdventDay();
-  const year = new Date().getUTCFullYear().toString();
-  const day = String(currentAdventDay);
-  const endSlug = getAotSlug({ year, day });
+  // TODO: Put leaderboard under event/[year]/leaderboard
+  // pass in year from slug
+  const year = new Date().getUTCFullYear();
+
+  // Get the starting id of this year (first aot was 2023)
+  const startId = (year - 2023) * 25 + 1;
```

Comment:
not following how this works 🤔

Reply:
The TrackChallenge table will have id's starting from 1 and auto incrementing for each challenge released each day of advent. So first year of advent of typehero was in 2023.

E.g. 2023's id's will always be 1-25, 2024 will be 26-50, etc

Reply:
that's not true though. we have other tracks outside of advent of typescript that could intertwine with the ranges

Reply:
Oh lol 😆 😭